Shared/Samples/Browse OGC API feature service/BrowseOGCAPIFeatureServiceView.swift

```diff
@@ -0,0 +1,256 @@
+            return renderer
+        }
+        
+        /// Create and load the OGC API features service from a URL.
+        /// - Parameter url: The URL of the OGC service.
+        /// - Returns: Returns a `OCGFeatureService` that has been loaded and initialized.
+        private func makeService(url: URL) async throws -> OGCFeatureService {
+            let service = OGCFeatureService(url: url)
+            try await service.load()
+            if service.loadStatus == .loaded,
+               let serviceInfo = service.serviceInfo {
+                featureCollectionInfos = serviceInfo.featureCollectionInfos
+                layerNames = featureCollectionInfos.map(\.title)
+            }
+            return service
+        }
+        
+        /// Loads OGC service for a URL so that it can be rendered on the map.
+        /// - Parameters:
+        ///   - mapProxy: Allows access to `MapView`
+        ///   - url: The URL of the OGC service.
+        func loadOGCFeatureData(mapProxy: MapViewProxy, url: URL?) async throws {
+            service = try await makeService(
+                url: url ?? .defaultServiceURL
+            )
+            try await displayLayer(
+                with: featureCollectionInfos[0],
+                proxy: mapProxy
+            )
+        }
+        
+        /// Updates the selected info property for the users selection.
+        /// - Parameter selection: String with the name of the selected layer to display.
+        func update(for selection: String) {
+            selectedInfo = featureCollectionInfos.first(where: {
+                $0.title == selection
+            })
+        }
+        
+        /// Load and display a feature layer from the OGC feature collection table.
+        /// - Parameter info: The `OGCFeatureCollectionInfo` selected by user.
+        func displayLayer(with info: OGCFeatureCollectionInfo, proxy: MapViewProxy) async throws {
+            let table = OGCFeatureCollectionTable(featureCollectionInfo: info)
+            // Set the feature request mode to manual (only manual is currently
+            // supported). In this mode, you must manually populate the table -
+            // panning and zooming won't request features automatically.
+            table.featureRequestMode = .manualCache
+            _ = try await table.populateFromService(using: queryParameters, clearCache: false)
+            if let extent = info.extent {
+                let featureLayer = FeatureLayer(featureTable: table)
+                if let geoType = table.geometryType {
+                    featureLayer.renderer = getRendererForTable(withType: geoType)
+                    map.addOperationalLayers([featureLayer])
+                    await proxy.setViewpointGeometry(extent, padding: 100)
+                    self.selectedInfo = info
+                }
+            }
+        }
+    }
+}
+
+private extension URL {
+    static let defaultServiceURL = URL(string: "https://demo.ldproxy.net/daraa")!
```

Comment:
I don't think this is needed anymore.

Reply:
You are right. The state variable's initial value should be enough.